tests/regression/roles/sonic_bgp_as_paths/defaults/main.yml

```diff
@@ -52,19 +57,22 @@ tests:
             - "101.101"
             - "201.201"
             - "301.301"
+          permit: False
         - name: test_2
           members:
             - '111\\:'
             - '11\\d+'
             - '113\\*'
             - '114\\'
+          permit: True
```

Comment:
In addition to the 'True' and 'False' test cases, it would be good to verify the ability to change the permit value from True to False, or vice versa.

Reply:
[bgp_as_path_log.pdf](https://github.com/ansible-collections/dellemc.enterprise_sonic/files/7499773/bgp_as_path_log.pdf)